src/aioswitcher/api/__init__.py

```diff
@@ -325,3 +469,264 @@ async def create_schedule(
+            if self.on_off_type and current_state and current_state != state:
+                # This is a toggle mode AC.
+                key.append("on_")
+
+            # for toggle mode AC - set state. for non toggel AC mode set state and turn
+            # it on.
+            if self.on_off_type or (not self.on_off_type and state == DeviceState.ON):
+                # Auto and Dry can sometimes have a FAN level and in other cases
+                # it might not have
+                if mode == ThermostatMode.AUTO or mode == ThermostatMode.DRY:
+                    key.append(BreezeRemote.MODE_TO_COMMAND[mode])
+                    key.append("_" + BreezeRemote.FAN_LEVEL_TO_COMMAND[fan_level])
+
+                    if swing == ThermostatSwing.ON:
+                        key.append("_d1")
+                    while len(key) != 0:
+                        if "".join(key) not in self._ir_wave_map:
+                            key.pop()
+                        else:
+                            break
+
+                if mode == ThermostatMode.COOL or mode == ThermostatMode.HEAT:
+                    key.append(BreezeRemote.MODE_TO_COMMAND[mode])
+                    key.append(str(target_temp))
+                    key.append("_" + BreezeRemote.FAN_LEVEL_TO_COMMAND[fan_level])
+                    if swing == ThermostatSwing.ON:
+                        key.append("_d1")
+
+                    while len(key) != 0:
+                        if "".join(key) not in self._ir_wave_map:
+                            key.pop()
+                        else:
+                            break
+
+        command = (
+            self._ir_wave_map["".join(key)]["Para"]
+            + "|"
+            + self._ir_wave_map["".join(key)]["HexCode"]
+        )
+        return SwitcherBreezeCommand(
+            "00000000" + hexlify(str(command).encode()).decode()
+        )
+
+    def _resolve_capabilities(self, ir_set):
+        """Parse the ir_set of the remote and build capability data struct."""
+        if ir_set["OnOffType"] == 1:
+            self.on_off_type = True
+
+        for wave in ir_set["IRWaveList"]:
+            key = wave["Key"]
+            try:
+                mode = BreezeRemote.COMMAND_TO_MODE[key[0:2]]
+                self.cap_modes.add(mode)
+            except KeyError:
+                pass
+
+            fan_level = re.match(r".+(f\d)", key)
+            if fan_level:
+                self.cap_fan_levels.add(
+                    BreezeRemote.COMMAND_TO_FAN_LEVEL[fan_level.group(1)]
+                )
+
+            temp = key[2:4]
+            if temp.isdigit():
+                temp = int(temp)
+                if temp > self.cap_max_temp:
+                    self.cap_max_temp = temp
+                if temp < self.cap_min_temp:
+                    self.cap_min_temp = temp
+
+            self.cap_swingable = not self.cap_swingable and "d1" in key
```

Comment:
```suggestion
            self.cap_swingable |= "d1" in key
```
Using not will set it back to false on next iteration

Reply:
Correct, However in my IRSet it looks different.

There are two keys FUN_d1 and FUN_d0

In Switcher app if I press the swing, it'll swing but it shows in the APP that the swing is Moving but after few seconds it shows Stop.

I believe for such AC the swing command should be handled differently and in a specific command.

Reply:
I guess I most go back with this to them and ask how it can be handled in single command.